test/libsolidity/semanticTests/constructor/no_callvalue_check.sol

```diff
@@ -9,17 +9,17 @@ contract B3 { constructor() payable {} }
 contract C {
 	function f() public payable returns (bool) {
 		// Make sure none of these revert.
-		new B1{value: 10}();
-		new B2{value: 10}();
-		new B3{value: 10}();
+		new B1{value: 10, salt: hex"00"}();
+		new B2{value: 10, salt: hex"01"}();
+		new B3{value: 10, salt: hex"02"}();
```

Comment:
Cases like this fail on EOF, because the default salt is `0` and therefore unsalted `new` always generates the same address. And deploying to the same address causes a revert unless the initcode is identical.